crimsobot/utils/games.py

```diff
@@ -13,6 +14,41 @@
 DiscordUser = Union[discord.User, discord.Member]
 
 
+def return_cball_answer(ctx: commands.Context) -> str:  # function to give first answer a ctx to work with
```

Comment:
nit: `return_cball_answer` is a bit of a weird method name, something like `get_crimsoball_answer` is probably better.

not a blocking suggestion though, this is fine.

Reply:
good idea; done in https://github.com/crimsobot/crimsoBOT/pull/73/commits/fb038dac1c73a155a20fdf5ca59e4f9fb582ac05